script.js

```diff
@@ -0,0 +1,62 @@
+//-------------------------------------------------------1
+
+// for (let index = 10; index <= 40; index++) {
+//     if(index % 5 == 0 && index % 3 == 0) {
+//         console.log(`${index} "Jackpot!"`);
+//     } else if(index % 5 == 0) {
+//         console.log(`${index} "This is divisible by 5"`);
+//     } else if (index % 3 == 0) {
+//         console.log(`${index} "This is divisible by 3"`);
+//     }else {
+//         console.log(index);
+//     }
+// }
+
+//-----------------------------------------------------2
+const body = document.querySelector('body'),
+      button = document.createElement('button');
+
+button.innerHTML = 'button';
+body.style.cssText = `
+    margin-top: 40px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+`;
+body.appendChild(button);
+
+function buttonClick() {
+    const h1 = document.createElement('h1');
+    h1.innerText = 'This is an h1 tag';
+    body.appendChild(h1);
+    button.style.backgroundColor = 'red';
+    button.removeEventListener('click', buttonClick);
+}
+
+button.addEventListener('click', buttonClick);
```

Comment:
Well done again! I have almost nothing to add, except that the button should become disabled in the function, and what you are doing is just removing the event listener. Now, this basically means that the button is now useless, but it is not disabled, as I can still click it. You should have used `button.disabled = true;`, and then the button would also become greyish, and you can no longer click it. 😉